the lume rockspec uses git:// as the protocol, which github does not support
anymore. there is also zero way to override this without a global git replace
insteadOf setting.. so we have to clone it by hand
diff --git a/Makefile b/Makefile
index dd10ece..f5a62fc 100644
--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ LUA_ROCKS ?= /usr/bin/luarocks-5.3
 DESTDIR=/usr/bin
 
 fetch-libraries:
-	$(LUA_ROCKS) install lume --local
+	@echo 123
 
 compile-lua:
 	$(MAKE) fetch-libraries
